src/fondant/core/schema.py

```diff
@@ -2,13 +2,26 @@
 and pipelines.
 """
 
+import os
 import re
 import typing as t
+from enum import Enum
 
 import pyarrow as pa
 
 from fondant.core.exceptions import InvalidTypeSchema
 
+
+class CloudCredentialsMount(Enum):
```

Comment:
I think this is only used by the runner SDK, so would move it there.

Reply:
Moved it to the compiler since this is where the extra volume mounting happens

Reply:
Great, I meant pipeline SDK anyway.